docs/dev-env-first-time-contributors.md

```diff
@@ -701,54 +701,10 @@ patching file bundler.rb
 
 See ["Possible testing issues"](testing.html#possible-testing-issues).
 
-### `provision.py` fails during `vagrant up` on Ubuntu
 
-If you see the following error after running  `vagrant up` :
+### `pip install` fails during `vagrant up` on Ubuntu
 
-```
-==> default: + pip install --no-deps --requirement /srv/zulip/requirements/py2_dev.txt
-==> default: Traceback (most recent call last):
-==> default:   File "/srv/zulip/tools/provision.py", line 243, in <module>
-==> default:     
-==> default: sys.exit(main())
-==> default:   File "/srv/zulip/tools/provision.py", line 182, in main
-==> default:     
-==> default: setup_virtualenv(PY2_VENV_PATH, DEV_REQS_FILE)
-==> default:   File "/srv/zulip/scripts/lib/setup_venv.py", line 47, in setup_virtualenv
-==> default:     
-==> default: do_setup_virtualenv(cached_venv_path, requirements_file, virtualenv_args or [])
-==> default:   File "/srv/zulip/scripts/lib/setup_venv.py", line 71, in do_setup_virtualenv
-==> default:     
-==> default: run(["pip", "install", "--no-deps", "--requirement", requirements_file])
-==> default:   File "/srv/zulip/zulip_tools.py", line 91, in run
-==> default:     
-==> default: raise subprocess.CalledProcessError(rc, args) # type: ignore # https://github.com/python/typeshed/pull/329
-==> default: subprocess
-==> default: .
-==> default: CalledProcessError
-==> default: : 
-==> default: Command '['pip', 'install', '--no-deps', '--requirement', '/srv/zulip/requirements/py2_dev.txt']' returned non-zero exit status 1
-```
-
-Error log might not look exactly same so look for these lines:
-
-```
-==> default: Traceback (most recent call last):
-==> default:   File "/srv/zulip/tools/provision.py", line 243, in <module>
-==> default:     
-==> default: sys.exit(main())
-==> default:   File "/srv/zulip/tools/provision.py", line 182, in main
-```
-
-and
-
-```
-==> default: raise subprocess.CalledProcessError(rc, args) # type: ignore # https://github.com/python/typeshed/pull/329
-==> default: subprocess
-==> default: .
-==> default: CalledProcessError
-```
-
-It's likely you are running a VM and it's ran out of memory. 
+Likely cause are:
 
-Check if you have alotted RAM according to [Requirements](dev-env-first-time-contributors.html#requirements), which is minimum 2GB. If not, go to your VM setting and increase the RAM then restart the VM.
+1. Networking Issues
+2. Insufficient RAM
```

Comment:
I understand that you were responding to Tim's comments when you cut this down, but it's okay -- in fact, preferred -- to spell this out a little more, and I think that you were right in your previous draft to mention a few details about the amount of RAM you're supposed to have.